trick-5.php

```diff
@@ -1,6 +1,13 @@
 <?php
 
-/* Insert Magic Code Here */
+$data = new class implements Countable {
```

Comment:
you could also just instantiate $data like that `$data = 'Hellm';`, but that raises a warning in php 7.2